index.html

```diff
@@ -8,62 +8,210 @@
-          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like
-                what you see and have a project you need coded, don't hesitate to contact me.</p>
-          <div class="icons-container">
-            <ul class="social-media-icons d-flex flex-center">
-              <li>
-                <a href="https://twitter.com/CynthiaInga_C">
-                  <img src="./image/icons/twitter-icon.png" alt="twitter">
-                </a>
-              </li>
-              <li>
-                <a href="https://www.linkedin.com/in/cynthia-inga7/">
-                  <img src="./image/icons/Linkedin-icon.png" alt="linkedin">
-                </a>
-              </li>
-              <li>
-                <a href="#">
-                  <img src="./image/icons/medium-icon.png" alt="medium">
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/cynthiainga">
-                  <img src="./image/icons/github-icon.png" alt="github">
-                </a>
-              </li>
-              <li>
-                <a href="https://angel.co/company/profile/">
-                  <img src="./image/icons/angellist-icon.png" alt="angellist">
-                </a>
-              </li>
+          </li>
+          <li class="section-1 d-flex flex-center">
+            <img
+              src="https://yourpng.com/public/img-category/cb-editing-2022-background-XmCcK34Om7ndx2HKxhlkDGlgxxcsbiV2.jpg"
+              alt=""/>
+            <p>Multi-Post Stories</p>
+            <p>Gain+Glory</p>
+            <ul>
+              <li>Ruby</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>html</li>
             </ul>
-          </div>
-          <div class="scroll-down-icon">
-            <a href="#">
-              <img src="./image/icons/croll.png" class="fa" alt="scroll-down" />
-            </a>
-          </div>
-        </section>
-      </main>
-    </body>
+            <div class="btn d-flex">
+              <button class="">See Project</button>
+            </div>
+            </li>
+          <li class="section-1 d-flex flex-center">
+            <img
+              src="https://yourpng.com/public/img-category/cb-editing-2022-background-XmCcK34Om7ndx2HKxhlkDGlgxxcsbiV2.jpg"
+              alt=""/>
+            <p>Multi-Post Stories</p>
+            <p>Gain+Glory</p>
+            <ul>
+              <li>Ruby</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <div class="btn d-flex">
+              <button class="">See Project</button>
+            </div>
+            </li>
+          <li class="section-1 d-flex flex-center">
+            <img
+              src="https://yourpng.com/public/img-category/cb-editing-2022-background-XmCcK34Om7ndx2HKxhlkDGlgxxcsbiV2.jpg"
+              alt=""/>
+            <p>Multi-Post Stories</p>
```

Comment:
- [ ] The project title (e.g., "Multi-Post Stories") uses a header tag hx, It may be the `h2` or `h3` tag. Kindly fix it.

Reply:
Already change